package.json

```diff
@@ -33,7 +33,7 @@
   "devDependencies": {
     "delete": "^1.1.0",
     "gulp-format-md": "^2.0.0",
-    "mocha": "^6.2.0"
+    "mocha": "^6.2.1"
```

Comment:
There's no need to update this in this PR.

Reply:
Reverted.